### PR TITLE
Extra unit tests

### PR DESCRIPTION
**Pull request by gauridhawan**
# Pull Request Template

## Checklist:

- [X] I have written test cases for this change, and my code passed the test
- [ ] I have updated documentation to reflect this change
- [X] I have attached this PR to an issue/user story

### CODE REVIEW

**Review by iLtc — APPROVED** [review]
LGTM, thanks!

File: tests/test_service_routes.py [hunk 1]
```diff
@@ -385,6 +385,31 @@ def test_update_shopcart_item(self):
...
+            "/shopcartitems/{}".format(1),
+            json=new_shopcart_item,
+            content_type="application/json",
+        )
+        self.assertEqual(resp.status_code, status.HTTP_404_NOT_FOUND)
+
+    def test_query_shopcart_item(self):
+        """ Query shopcart item list without any query string """
+        test_shopcart = ShopcartFactory()
+        resp = self.app.post("/shopcarts", json={"user_id": test_shopcart.user_id}, content_type="application/json")
```
Comment (KHafner) [inline tests/test_service_routes.py:404]: I think this line will give an error in pylint for being too long

File: tests/test_service_routes.py [hunk 2]
```diff
@@ -501,6 +526,18 @@ def test_delete_shopcart_items(self):
 
         self.assertEqual(len(ShopcartItem.all()), 0)
 
+    def test_post_request_without_content_type(self):
+        """POST Request Without content_type"""
+        resp = self.app.post("/shopcarts")
+        self.assertEqual(resp.status_code, status.HTTP_415_UNSUPPORTED_MEDIA_TYPE)
+        self.assertIn(b'Unsupported media type', resp.data)
+
+    def test_post_request_with_wront_content_type(self):
```
Comment (KHafner) [inline tests/test_service_routes.py:535]: minor typo here :)

File: tests/test_service_routes.py (hunk 1 above)
Comment (iLtc) [reply tests/test_service_routes.py:404]: I didn't realize we need to run pylint for test files. Should I do a bulk update to fix other syntax issues in tests?

Comment (KHafner) [reply tests/test_service_routes.py:404]: oh I thought it was for all the .py files. the score is above 9.0 for them right now

**Review by jules203 — APPROVED** [review]
Looks good!

**Pull request merged**